cmd/goal/account.go

```diff
@@ -129,11 +127,16 @@ func init() {
 	changeOnlineCmd.Flags().BoolVarP(&online, "online", "o", true, "Set this account to online or offline")
 	changeOnlineCmd.MarkFlagRequired("online")
 	changeOnlineCmd.Flags().Uint64VarP(&transactionFee, "fee", "f", 0, "The Fee to set on the status change transaction (defaults to suggested fee)")
-	changeOnlineCmd.Flags().Uint64VarP(&statusChangeFirstRound, "firstRound", "", 0, "FirstValid for the status change transaction (0 for current)")
-	changeOnlineCmd.Flags().Uint64VarP(&statusChangeValidRounds, "validRounds", "v", 0, "The validity period for the status change transaction")
+	changeOnlineCmd.Flags().Uint64VarP(&firstValid, "firstRound", "", 0, "FirstValid for the status change transaction (0 for current, deprecated)")
+	changeOnlineCmd.Flags().Uint64VarP(&firstValid, "firstvalid", "", 0, "The first round where the status change transaction may be committed to the ledger (0 for current)")
+	changeOnlineCmd.Flags().Uint64VarP(&numValidRounds, "validRounds", "", 0, "The validity period for the status change transaction (deprecated)")
+	changeOnlineCmd.Flags().Uint64VarP(&numValidRounds, "validrounds", "v", 0, "The validity period for the status change transaction")
+	changeOnlineCmd.Flags().Uint64Var(&lastValid, "lastvalid", 0, "The last round where the transaction may be committed to the ledger")
 	changeOnlineCmd.Flags().StringVarP(&statusChangeLease, "lease", "x", "", "Lease value (base64, optional): no transaction may also acquire this lease until lastvalid")
 	changeOnlineCmd.Flags().StringVarP(&statusChangeTxFile, "txfile", "t", "", "Write status change transaction to this file")
 	changeOnlineCmd.Flags().BoolVarP(&noWaitAfterSend, "no-wait", "N", false, "Don't wait for transaction to commit")
+	changeOnlineCmd.Flags().MarkDeprecated("firstRound", "FirstValid for the status change transaction (0 for current)")
+	changeOnlineCmd.Flags().MarkDeprecated("validRounds", "The validity period for the status change transaction")
```

Comment:
I didn't find the documentation, but it seems like the depracation string is probably something about why it is deprecated? i.e. "Please use --validrounds instead."

Reply:
Our are right, fixed